docs/sphinx/users/micromanager/index.txt

```diff
@@ -0,0 +1,39 @@
+Micro-Manager
+=============
+
+- Micromanager offers the functionality for saving the acquired images in tiff/ome-tiff format. 
+- Based on the mode of saving and the configuration settings, the acquired image can be saved 
+  with/without a companion file (metadata.txt).
+
+.. figure:: /images/micromanager_saving.png
+    :width: 180px
```

Comment:
This screenshot could do with being bigger, I can't read the text on http://www.openmicroscopy.org/site/support/bio-formats5.2-staging/users/micromanager/index.html
Maybe try 50% instead?